strawberry_django/type.py

```diff
@@ -1,13 +1,15 @@
 import strawberry
 import typing
 from .types import get_model_fields, update_fields
+from .utils import deprecated
 
 _type = type
 
-def type(model, *, fields=None, types=None, is_update=False, **kwargs):
+def type(model, *, fields=None, types=None, **kwargs):
     def wrapper(cls):
         is_input = kwargs.get('is_input', False)
-        model_fields = get_model_fields(cls, model, fields, types, is_input, is_update)
+        is_partial = kwargs.pop('is_partial', False)
```

Comment:
```suggestion
        is_partial = kwargs.pop('partial', False)
```

what about just `partial`? :)

Reply:
that's even better :). why do we have is_input, instead if input? I just followed the same format herem. is that to avoid collision with internals?

Reply:
Renamed

Reply:
good question! yes to prevent clashing with internals, but also `is_input` is not really exposed to end users (and I think it might go away in future) :)